homeassistant/components/miele/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "miele",
+  "name": "Miele",
+  "codeowners": ["@astrandb"],
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
+  "documentation": "https://www.home-assistant.io/integrations/miele",
+  "iot_class": "cloud_push",
+  "loggers": ["pymiele"],
+  "quality_scale": "bronze",
+  "requirements": ["pymiele==0.3.1"],
+  "single_config_entry": true
```

Comment:
why the single config entry?

Reply:
There is no way to identify the account from API data. The access_token is not jwt and I dont know if it can be parsed. I suggest that we keep it as single account for the time being.